spec/presenters/tslr_claim_csv_row_spec.rb

```diff
@@ -14,7 +14,6 @@
     let(:claim) do
       create(:tslr_claim, :submittable,
         date_of_birth: Date.parse(date_of_birth),
-        mostly_teaching_eligible_subjects: mostly_teaching_eligible_subjects == "Yes",
```

Comment:
Should we also remove the `let` associated with this?

Reply:
👍🏽